src/Ztm.Zcoin.NBitcoin/ZcoinTransaction.cs

```diff
@@ -19,6 +20,43 @@ public ZcoinTransaction(ZcoinConsensusFactory consensusFactory)
         }
         #pragma warning restore CS0618
 
+        public bool IsZerocoinSpend
+        {
+            get
+            {
+                return Inputs.Any(txIn => txIn.IsZerocoinSpend());
```

Comment:
I think we should use `All` instead. I know in Zcoin Core we use `Any` but in reality `IsZerocoinSpend` should return `true` only if it is valid Zerocoin Spend. That mean all inputs must be Zerocoin spend.